pallets/parachain-staking/src/migrations/v5.rs

```diff
@@ -31,10 +41,30 @@ pub(crate) fn migrate<T: Config>() -> Weight {
 	// Kill selected candidates list
 	old::SelectedCandidates::<T>::kill();
 
-	// count candidates
-	let counter: u32 = CandidatePool::<T>::iter().fold(0, |acc, _| acc.saturating_add(1));
+	// migrate CandidatePool to TopCandidates
```

Comment:
Q: There's no added weight for all these reads/writes that we do now?

Reply:
good point completely forgot the weights...

Reply:
should be fixed